src/libs/API/parameters/OpenReportParams.ts

```diff
@@ -21,6 +21,7 @@ type OpenReportParams = {
      * If this flag is set to true, api will return oldest first starting from the beginning of report.
      */
     useTableReportView?: boolean;
+    transactionID?: string;
```

Comment:
Since we are also creating a money request preview for the transaction we will want to also pass an optimistic `moneyRequestPreviewReportActionID`. Can we also put some comments here to explain the purpose of these both?

Reply:
Comments here would be good.